src/screening/eid/DnaPcrForm.jsx

```diff
@@ -6,30 +6,28 @@ import { Grid, Row, FormGroup, ControlLabel, Col } from 'react-bootstrap';
 import { CONCEPTS, FORM_NAMES, FORM_ANSWERS } from "../../constants";
 import './styles/dna-pcr-form.css';
 
-
 class DnaPcrForm extends React.PureComponent {
 
   componentDidUpdate(prevProps) {
     // this clears out form values when the "bled" question is changed
     if (typeof this.props.bled !== 'undefined' && this.props.bled !== prevProps.bled) {
       if (this.props.bled === CONCEPTS.True.uuid) {
-        this.props.dispatch(change(FORM_NAMES.eidFormName, formUtil.obsFieldName('dna-pcrdr-reason-no-sample', CONCEPTS.ReasonForNoSample.uuid), null));
-        this.props.dispatch(untouch(FORM_NAMES.eidFormName, formUtil.obsFieldName('dna-pcrdr-reason-no-sample', CONCEPTS.ReasonForNoSample.uuid)));
+        this.props.dispatch(change(this.props.formInstanceId, formUtil.obsFieldName('dna-pcrdr-reason-no-sample', CONCEPTS.ReasonForNoSample.uuid), null));
+        this.props.dispatch(untouch(this.props.formInstanceId, formUtil.obsFieldName('dna-pcrdr-reason-no-sample', CONCEPTS.ReasonForNoSample.uuid)));
       }
```

Comment:
Oops... I had forgotten to change these when I switched to using the form instance ID... nice catch..